packages/terra-functional-testing/package.json

```diff
@@ -47,12 +47,12 @@
   "dependencies": {
```

Comment:
Should we bump the supported Node versions [here](https://github.com/cerner/terra-toolkit/pull/674/files#diff-72c2847b50d4152bd910ef25c33dafbd31a04d507329a333036e6226e5e8ef71R10) to include 14?

`"node": ">=10 <15"`

Reply:
Actually we should exclude Node 10 since V7 [dropped Node 10 support](https://webdriver.io/blog/2021/02/09/webdriverio-v7-released/#drop-node-v10-support).

`"node": ">=12 <15"`

Reply:
Good point. Updated.